packages/web/components/Player/PlayerContacts.tsx

```diff
@@ -15,83 +14,59 @@ type Props = {
 export const PlayerContacts: React.FC<Props> = ({
   player,
   disableBrightId = true, // TODO: enable after fixing issue #1068
-}) => {
-  const [copied, handleCopy] = useCopyToClipboard();
-  return (
-    <Wrap justify="center">
-      {player?.accounts?.map((acc) => {
-        switch (acc.type) {
-          case 'TWITTER': {
-            const link = `https://twitter.com/${acc.identifier}`;
-            return (
-              <WrapItem key={link}>
-                <Button
-                  as="a"
-                  href={link}
-                  target="_blank"
-                  rel="noreferrer noopener"
-                  size="xs"
-                  colorScheme="twitter"
-                  color="white"
-                  leftIcon={<FaTwitter />}
-                >
-                  {acc.identifier}
-                </Button>
-              </WrapItem>
-            );
-          }
-          case 'GITHUB': {
-            const link = `https://github.com/${acc.identifier}`;
-            return (
-              <WrapItem key={link}>
-                <Button
-                  as="a"
-                  href={link}
-                  target="_blank"
-                  rel="noreferrer noopener"
-                  size="xs"
-                  colorScheme="blackAlpha"
-                  backgroundColor="black"
-                  color="white"
-                  leftIcon={<FaGithub />}
-                >
-                  {acc.identifier}
-                </Button>
-              </WrapItem>
-            );
-          }
-          default: {
-            return null;
-          }
+}) => (
+  <Wrap justify="center" pointerEvents="all" zIndex={1000}>
+    {player?.accounts?.map((acc) => {
+      switch (acc.type) {
+        case 'TWITTER': {
+          const link = `https://twitter.com/${acc.identifier}`;
```

Comment:
As with the `IconButton`s above, I'd move the repeated code out of the `switch` and just set an object. Something like:

```javascript
    {player?.accounts?.map((acc) => {
      let info;
      switch (acc.type) {
        case 'TWITTER': {
          info = {
            link: `https://twitter.com/${acc.identifier}`,
            label: 'Twitter',
            icon: <FaTwitter/>,
          }
          break
        }
        ⋮
```

In fact, since these buttons are identical to the ones above, I'd just define a component for it:

```javascript
<LinkIconButton {...info}/>
```